Automattic/wp-calypso
# Launchpad: Removes automatic activation of the Checklist

Related to #82669

## Proposed Changes

* Activating the checklist means that it will be displayed on the navigator as the primary checklist. Every time the user interacts with a checklist, it will be defined as the active checklist in the navigator,  and we'll display the referred checklist in the Navigator component. The intention of this functionality is to keep track of which task lists the user is working on.
* But for the demo, we decided to allow only the Launchpad on the Customer Home to activate the checklist since we are not handling more than one in the navigator yet.
* This PR depends on https://github.com/Automattic/wp-calypso/pull/82669 to add the `DefaultWiredLaunchpad` component on the Customer Home so that we can add the `shouldActivateChecklist = true`. 

## Testing Instructions



* Use the Calypso Live link or apply this PR to your local environment
* On a site where you didn't complete the Earn launchpad, navigate to `/earn/:siteSlug`.
* Click on a task in the Earn launchpad and ensure **no** request is made to `/wpcom/v2/sites/:siteSlug/launchpad/navigator?_envelope=1` with the payload `{"active_checklist_slug": "earn"}`.
* Go to the Customer Home page on a site with any launchpad available.
* Click on a task, and make sure a request is made to `/wpcom/v2/sites/:siteSlug/launchpad/navigator?_envelope=1` with the payload `{"active_checklist_slug": "checklist-name"}`
* Be aware that the request will not go through if the active_checklist_slug is already the same as above. You can make a request manually on the Developer Console to update it with a different checklist, refresh the page, and try again.

## Pre-merge Checklist
